src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -56,7 +58,13 @@ export default function QuickTrace({event, location, organization}: Props) {
             tooltipText={t('A minified version of the full trace.')}
             bodyText={body}
             subtext={
-              trace === null ? '\u2014' : t('Trace ID: %s', getShortEventId(traceId!))
+              traceId === null ? (
+                '\u2014'
+              ) : (
+                <Link to={traceTarget}>
+                  {t('Trace ID: %s', getShortEventId(traceId))}
```

Comment:
Do we translate dynamic content?

Reply:
What do you mean? This should only be translating `Trace ID`.

Reply:
oh, nvm I wasn't aware that jed ignored the `%s`, should have been obvious since there is multiple params to the function.